doc/rvv-intrinsic-spec.adoc

```diff
@@ -626,9 +626,9 @@ unsigned __riscv_vlenb();
 
 === Uninitialized variables with RVV types and `vundefined`
 
-Object created in the declaration of an RVV type represents a scalable-sized object that corresponds to a memory address. The default values inside such uninitialized values, and also to variables being assigned to the `vundefined` intrinsics represent non-deterministic values. Users should be aware to not rely on the result of these values.
+Variables created in the declaration of an RVV type represents a scalable-sized object that corresponds to a memory address. The default values inside such uninitialized variables, and also to variables being assigned to the `vundefined` intrinsics, represent non-deterministic values. Users should be aware to not rely on the result of these values.
```

Comment:
* It's not clear what "corresponds to a memory address" means here.
* It should be clarified that this also applies to masked-out elements (with "ma") and tail elements (with "ta"): these also become uninitialized values.